### ISSUE TITLE
learning rate for 4 gpu on imagenet dataset

### ISSUE THREAD

**Issue by rajatsaini0294** [facebookresearch/ResNeXt#13]
What lr should I choose for imagenet training if I use 4 GPUs instead of 4 with batch size of 32 per gpu?
